changelog.txt

```diff
@@ -1,3 +1,8 @@
+v1.2.2 (2020-10-04)
+- Fix: removed dependencies on requests, tmdbsimple, and trakt modules
```

Comment:
I don't think these should be `Fix`?

Reply:
I was trying to keep the changelog to the same format, and there were only Features and Fixes. Features felt like user facing things, so I used Fixes. Could we just list it as Change?